README.md

```diff
@@ -99,6 +99,12 @@ apisix-website
     └── docusaurus.config.js
 ```
 
+#### IDE configuration and plug-in recommendations
```

Comment:
```suggestion
#### IDE configuration and plugin recommendations
```

Reply:
Let's add both forms, the document is to declare what helps to configure, and .config is the action of directly configuring :)